mlrun/config.py

```diff
@@ -112,7 +112,7 @@
             "missing_runtime_resources_debouncing_interval": None,
             # max number of parallel abort run jobs in runs monitoring
             "concurrent_abort_stale_runs_workers": 10,
-            "list_runs_time_period_in_days": 7,  # days
+            "list_runs_time_period_in_hours": 24 * 3,  # 3 days
```

Comment:
Is that a 1.7 feature? If so ok to break if not need to slowly deprecate or so.

Reply:
I'm not sure if deprecation is really needed because it's only used in server side and is not part of the client spec.
But I think I will revert this change to days because the latest state change allows for more grace